chat/rich-links/src/RichLink/RichLink.types.ts

```diff
@@ -0,0 +1,61 @@
+import { DarkModeProps, HTMLElementProps } from '@leafygreen-ui/lib';
+
+import { RichLinkBadgeProps } from './RichLinkBadge/RichLinkBadge.types';
+import { RichLinkVariantName } from './RichLinkVariants';
+
+export interface BaseRichLinkProps extends DarkModeProps {
+  /**
+   * The text that shows on the rich link
+   */
+  text: string;
```

Comment:
I think this might be better as `children` instead of `text.` That's the pattern we use for the `link` component. [Material UI](https://mui.com/material-ui/react-link/#basic-links) also uses this pattern. So, instead of passing the string to the `text` prop, we pass it as children.